src/btree/bt_split.c

```diff
@@ -31,6 +31,28 @@ typedef enum {
 } WT_SPLIT_ERROR_PHASE;
 
 /*
+ * __page_split_timing_stress --
+ *  Optionally add delay to simulate the race conditions in
+ *  page split for debug purposes. The purpose is to uncover
+ *  the race conditions in page split.
+ */
+static void
+__page_split_timing_stress(WT_SESSION_IMPL *session)
+{
+	WT_CONNECTION_IMPL *conn;
+
+	conn = S2C(session);
+
+	/*
+	 * We only want to sleep if the diagnostics flag is set
```

Comment:
This comment is out of date - and we also use single line form for comments that fit onto a single line (unless it's for a function header).